Prepare for Package Release

mdhorn opened the pull request:
Changes proposed in this pull request:
- Install launch, rviz, and config files.
- Migrate package.xml to format 2.
- Update the maintainers
- Update the package dependencies

reaganlo commented on realsense_navigation/CMakeLists.txt:
```diff
@@ -9,3 +9,18 @@ find_package(catkin REQUIRED)
+# Install config files
+install(DIRECTORY config/
+    DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/config
+)
```
Do we need to install the package.xml too?

mdhorn replied:
The package.xml is automatically placed in the install directory by catkin.

reaganlo replied:
Thanks for clarifying.

kevincwells commented on realsense_navigation/package.xml:
```diff
@@ -1,23 +1,21 @@
+
+  <exec_depend>roslaunch</exec_depend>
+  <exec_depend>realsense_camera</exec_depend>
+  <exec_depend>rgbd_launch</exec_depend>
```
Do we need this exec_depend on rgbd_launch?  Shouldn't we have an exec_depend on turtlebot, since we directly include launch files in that package?

mdhorn replied:
Yes, we should replace rgbd_launch with packages we are directly referencing in our launch files; my mistake as I was modeling after the [ros_astra_launch](https://github.com/orbbec/ros_astra_launch) example which only used 'rgbd_launch'.
We seem to have a long list:
`turtlebot_bringup, turtlebot_navigation, map_server, rviz, rosbag, amcl, gmapping`
Does this seem to be complete?

reaganlo commented on realsense_navigation/package.xml:
```diff
@@ -1,23 +1,21 @@
-  <run_depend>roscpp</run_depend>
-  <run_depend>rospy</run_depend>
+
+  <exec_depend>roslaunch</exec_depend>
```
Reading about "run_depend", it says in format 2, it should be replaced by two tags.
"build_export_depend"
"exec_depend"

mdhorn replied:
Are you referring to the <exec_depend>? <exec_depend> is a new tag in format 2.
`If the dependency is only used at run-time, only the <exec_depend> is needed.`

reaganlo replied:
Thanks for clarifying.

reaganlo commented:
Approved once all the exec_depends are identified and updated.

mdhorn committed:
Prepare for Package Release

Add installation steps for all files.

Update package.xml to version 2.
Updates to the maintainers and dependencies.

kevincwells commented on realsense_navigation/package.xml:
```diff
@@ -1,23 +1,27 @@
-  <run_depend>rospy</run_depend>
+
+  <exec_depend>roslaunch</exec_depend>
+  <exec_depend>realsense_camera</exec_depend>
```
Correct me if I'm wrong, but I think we don't actually have a direct dependency on realsense_camera package. We have a dependency on turtlebot_bringup, and THAT will have a dependency on realsense_camera (because it launches the 3d sensor).

Otherwise this list looks accurate and complete.

mdhorn replied:
@kevincwells Considering we have the realsense_navigation package in the same repository and we know it will indirectly require the realsense_camera package, are you opposed to leaving that dependency?

kevincwells replied:
I'm fine with leaving it in.

I approve this request.
